contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -1221,7 +1221,7 @@ WHEN t8.oid > 0 THEN CAST(t6.relname AS sys.sysname)
 ELSE CAST(SUBSTRING(t6.relname,1,LENGTH(t6.relname)-32-LENGTH(t1.relname)) AS sys.sysname) 
 END AS INDEX_NAME,
 CASE
-WHEN t7.starelid > 0 THEN CAST(0 AS smallint)
+WHEN t1.relkind = 'i' THEN CAST(0 AS smallint)
```

Comment:
I think it should be `WHEN t1.relkind != 'i' THEN CAST(0 AS smallint)` , isn't it?
Because in case of index, it should go to the else block.

Reply:
Removed it. It was redundant condition because type = 0 incase of table was already handled by 1st part of union query.